controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -168,6 +177,7 @@ func (r *StorageClassRequestReconciler) SetupWithManager(mgr ctrl.Manager) error
 			predicate.GenerationChangedPredicate{},
 		)).
 		Watches(&rookCephv1.CephBlockPool{}, enqueueForOwner).
+		Watches(&rookCephv1.CephBlockPoolRadosNamespace{}, enqueueForOwner).
```

Comment:
quick check, do we already have rbac for this?

Reply:
I dunno, why weren't we using `Owns` for CephBlockPools in the first place?  I was just following convention since it seems largely inconsequential to the feature behavior.

Reply:
From my point of view, both need to be using the `Owns` watch. 
The fact that  the block pool watch slipped through our review or that for that specific resource, there are abnormal conditions does not mean we need to repeat it without explaining.

Reply:
....okay so it turns out we are *almost* using `Owns()`, just obliquely. From https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.17.2/pkg/builder#Builder.Owns :

> By default, this is the equivalent of calling Watches(object, handler.EnqueueRequestForOwner([...], ownerType, OnlyControllerOwner())). 

...and indeed enqueueForOwner is almost an exact a wrapper on that. This is likely because we weren't setting OwnerRefs at the time this was developed.  Thus, from a technical perspective, this is *now* viable and I will make this change.